components/developer/ant/Makefile

```diff
@@ -77,3 +80,7 @@ build:	$(BUILD_32)
 install: $(INSTALL_32)
 
 test:  $(TEST_32)
+
+REQUIRED_PACKAGES += developer/java/openjdk8
+REQUIRED_PACKAGES += runtime/java/openjdk8
+REQUIRED_PACKAGES += library/java/commons-bsf
```

Comment:
If ant requires commons-bsf, please, split PR in two, so that common-bsf could be imported first